api/endpoint-tests/endpoint.sh

```diff
@@ -4,24 +4,22 @@ rm ./endpoint-tests/endpoint-data.json
 
 export NODE_ENV=test
 export API_URL=http://localhost:8081
-export COMPOSE_PROJECT_NAME=api
 
 unset DEV_DB_NAME
 
 echo "[]" > endpoint-data.json
 
-docker-compose -f ../docker-compose.endpoint-tests.yml up -d
+docker-compose -f ../docker-compose.endpoint-tests.yml -p api up -d
 
-docker-compose -f ../docker-compose.endpoint-tests.yml exec api-for-testing yarn run migrate
-docker-compose -f ../docker-compose.endpoint-tests.yml exec api-for-testing yarn run seed
-docker-compose -f ../docker-compose.endpoint-tests.yml exec api-for-testing yarn run test-endpoints $@
+docker-compose -f ../docker-compose.endpoint-tests.yml -p api exec -e LOG_LEVEL=verbose api-for-testing yarn run migrate
```

Comment:
Are these verbose logs left in intentionally or were they added for troubleshooting and should be removed before merge?

Reply:
it was intentional because I changed the level of a lot of the print statements that were being displayed to the dev running it